.github/workflows/flake8.yml

```diff
@@ -0,0 +1,66 @@
+name: Python code quality check
+
+# Controls when the action will run.
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  # This workflow contains two job called "flake8-grass7-scripts" and "flake8-gui-wxgui-scripts"
+
+  flake8-grass7-scripts:
+    runs-on: ubuntu-20.04
+
+    strategy:
+      matrix:
+        directory:
+        - grass7/db
+        - grass7/display
+        - grass7/general
+        - grass7/imagery
+        - grass7/misc
+        - grass7/raster
+        - grass7/raster3d
+        - grass7/temporal
+        - grass7/tools
+        - grass7/vector
+      fail-fast: false
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Install
+      run: |
+        python -m pip install --upgrade pip
+        pip install flake8==3.8.0
+    - name: Run Flake8 in ${{ matrix.directory }}
+      run: |
+        flake8 --config=../.flake8 --count --statistics --show-source --jobs=$(nproc) ${{ matrix.directory }}
```

Comment:
You removed the `cd ...` but left `../.flake8` here. Consequently, `../` goes outside of the cloned source code. This is normal Bash, just try the same command locally and see how it behaves.

Reply:
Fixed